qa/TL3_EfficientDet_convergence/test_tensorflow.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash -e
+
+function CLEAN_AND_EXIT {
+    exit $1
+}
+
+cd /opt/dali/docs/examples/use_cases/tensorflow/efficientdet
```

Comment:
I suggest using `pushd` and `popd`. `cd` might fail when we decide to run this script from another script

Reply:
Changed.